src/main/java/jenkins/plugins/publish_over_ssh/BapSshHostConfiguration.java

```diff
@@ -49,39 +51,53 @@
 import com.jcraft.jsch.SftpException;
 
 @SuppressWarnings("PMD.TooManyMethods")
-public class BapSshHostConfiguration extends BPHostConfiguration<BapSshClient, BapSshCommonConfiguration>
+public class BapSshHostConfiguration extends BPHostConfiguration<BapSshClient, BapSshCommonConfiguration> 
                                                                                         implements Describable<BapSshHostConfiguration> {
 
+    static final String LOCALHOST = "127.0.0.1";
     private static final long serialVersionUID = 1L;
     public static final int DEFAULT_PORT = 22;
     public static final int DEFAULT_TIMEOUT = 300000;
     public static final String CONFIG_KEY_PREFERRED_AUTHENTICATIONS = "PreferredAuthentications";
     private static final Log LOG = LogFactory.getLog(BapSshHostConfiguration.class);
+    public static final String DEFAULT_JUMP_HOST = "";
 
     private int timeout;
     private boolean overrideKey;
     private boolean disableExec;
     private final BapSshKeyInfo keyInfo;
+    private String jumpHost;
 
     // CSOFF: ParameterNumberCheck
     @SuppressWarnings("PMD.ExcessiveParameterList") // DBC for you!
     @DataBoundConstructor
     public BapSshHostConfiguration(final String name, final String hostname, final String username, final String encryptedPassword,
-                                   final String remoteRootDir, final int port, final int timeout, final boolean overrideKey,
```

Comment:
Sorry, I should have said to use DataBoundSetter instead of deprecating the constructor.